<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>8 puzzle </string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="pola">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>192</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="pole_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_2">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_3">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>0</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_6">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>64</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_5">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>64</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>64</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_8">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>128</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pole_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>128</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>101</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="przycisk_hint">
         <property name="text">
          <string>Hint</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="przycisk_solve">
         <property name="text">
          <string>Solve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randomize">
         <property name="text">
          <string>Randomize</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGra">
    <property name="title">
     <string>Gra</string>
    </property>
    <addaction name="actionWczytaj_gr"/>
    <addaction name="actionZapisz_rozwi_zanie"/>
   </widget>
   <addaction name="menuGra"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWczytaj_gr">
   <property name="text">
    <string>Wczytaj grę</string>
   </property>
  </action>
  <action name="actionZapisz_rozwi_zanie">
   <property name="text">
    <string>Zapisz rozwiązanie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
